### DISCUSSION TITLE
Connect Deye hybrid 3 phase inverter to a JBD BMS

### DISCUSSION THREAD

**Discussion by goofybms** [Louisvdw/dbus-serialbattery#429] [Q&A] [answered]
Dear community. I am a novice in terms of RS485, Can Bus etc, but need to know if the mentionned Inverter can communicate with the JBD BMS. Here is the Link:  https://fr.aliexpress.com/item/1005005061460783.html?spm=a2g0o.productlist.main.3.2e8e37e43LSdA1&algo_pvid=fe4da7ad-2fe1-4c0f-a117-8d544999b5c8&algo_exp_id=fe4da7ad-2fe1-4c0f-a117-8d544999b5c8-1&pdp_ext_f=%7B%22sku_id%22%3A%2212000031491252485%22%7D&pdp_npi=2%40dis%21EUR%21157.14%21111.57%21%21%21%21%21%40210217c716743613810921954d06bc%2112000031491252485%21sea&curPageLogUid=GWLkeeSmF7yz

Is there any software that needs to be uploaded? Any help will be appreciated.

Kind regards

**Comment by Louisvdw** [comment]
I don't think the Deye has a controller that you can add drivers too. 
So that means the battery needs to be what the firmware can handle. Thus if it does not spesificly say it is supported, then it will not work.

**Comment by goofybms** [comment, marked as answer]
Dear Louis
Since i wrote the message on github, i found out that the Seplos bms works
with the Deye inverter from Andy from off grid garage and also from a
french guy who confirmed it too. Just waiting for the new confirmation from
my client to order both: inverter+seplos bms

Thank you for your help

Marcel
